packages/paste-core/components/data-grid/src/DataGridHeaderSort.tsx

```diff
@@ -41,12 +45,22 @@ export const DataGridHeaderSort: React.FC<DataGridHeaderSortProps> = ({
   direction,
   onClick,
   element = 'DATA_GRID_HEADER_SORT',
+  i18nAscendingLabel = 'Sort ascending',
+  i18nDescendingLabel = 'Sort descending',
+  i18nUnsortedLabel = 'Unsorted',
   ...props
 }) => {
+  const i18nLabelDirectionMap = {
+    ascending: i18nAscendingLabel,
+    descending: i18nDescendingLabel,
+    none: i18nUnsortedLabel,
+  };
+
   return (
     // @ts-ignore fixme when button accepts element
```

Comment:
Suggestion: Mind removing this ts-ignore whilst your at it? I think button should be accepting an element by now. Just a little quick win tidy up